platform-coop-toolkit/coop-map-directory-index
# feat: add Type.icon

## Description

This PR adds `icon` to the `Type` model so that a Pinecone icon slug (see #95) can be assigned to each of the four primary Organization types:

- Cooperative
- Potential cooperative
- Shared platform
- Supporting organization

## Steps to test

1. Check out branch.
2. Run migrations.
3. Navigate to Types in the admin UI.

**Expected behavior:** There's a new `CharField` labelled `Icon`.

## Additional information

Not applicable.

## Related issues

Not applicable.
